build.gradle.kts

```diff
@@ -47,3 +50,20 @@ tasks.register("runIde") {
         throw GradleException("Use project specific runIde command, i.e. :jetbrains-core:runIde, :intellij:runIde")
     }
 }
+
+if (idea.project != null) { // may be null during script compilation
+    idea {
+        project {
+            settings {
+                taskTriggers {
+                    afterSync(":sdk-codegen:generateSdks")
```

Comment:
would be nice to mark the source sets as generated roots as well

Reply:
Ya, still cant get that to work for some reason...